bench/ndarray/fancy_index.py

```diff
@@ -136,16 +137,17 @@ def timer(arr):
         error_kw=dict(lw=2, capthick=2, ecolor='k'))
         labs+=label
 
-filename = f"results{labs}{NDIMS}D" + "sparse" if SPARSE else f"results{labs}{NDIMS}D"
+filename = f"{labs}{NDIMS}D" + "sparse" if SPARSE else f"{labs}{NDIMS}D"
+filename+=blosc2.__version__.replace('.','_')
```

Comment:
PEP8: always surround operands (`+=`) between spaces. Are you using ruff pre-commit?